### ISSUE TITLE
(aws-cdk): Parallel deployment of one stack to multiple regions fails

### ISSUE THREAD

**Issue by falnyr** [aws/aws-cdk#16357]
## :question: General Issue
I'm trying to deploy one stack multiple times in parallel to different regions. As per the screenshot below:
![image](https://user-images.githubusercontent.com/6673296/131936929-56d93bea-af9d-48fb-8632-662687188fdd.png)

In the pipeline, I'm getting a following error:
```
❌  FoundationsDev failed: AlreadyExistsException: ChangeSet cdk-deploy-change-set cannot be created due to a mismatch with existing attribute Description
```
There seems to be [another issue](https://github.com/aws/aws-cdk/issues/14316) already, but without resolution.

### The Question
What would be a good way to orchestrate the deployments of the same stack across multiple regions?

### Environment

  - **CDK CLI Version:** 1.118.0 
  - **Module Version:** 1.118.0
  - **Node.js Version:** 16.3.0
  - **OS:** Ubuntu
  - **Language (Version):** TypeScript (3.9.10)

### Other information
This is the Github Actions job I'm running:
```yaml
  dev-canada:
    name: 'Deploy: Dev - Canada'
    needs: [get-latest-release]
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@master
      - name: Configure AWS Credentials
        uses: aws-actions/configure-aws-credentials@v1
        with:
          aws-access-key-id: ${{ secrets.DEV_AWS_ACCESS_KEY_ID }}
          aws-secret-access-key: ${{ secrets.DEV_AWS_SECRET_ACCESS_KEY }}
          aws-region: ca-central-1
      - uses: actions/setup-node@v2
        with:
          node-version: '16.3.0'
      - run: npm install -g aws-cdk@1.118.0
        name: Install AWS CDK
      - run: npm install
        name: Install dependencies
      - name: CDK Deploy
        run: cdk deploy FoundationsDev -c version=${{ needs.get-latest-release.outputs.release-output }} --require-approval never
  # ...
  # dev-sydney:
  # dev-virginia:
  # ...
```

**Comment by skinny85** [comment]
Hey @falnyr ,

`cdk deploy` takes a `--change-set-name` option:

```
$  npm run cdk deploy -- --help
cdk deploy [STACKS..]

Deploys the stack(s) named STACKS into your AWS account

Options:
  -a, --app                  REQUIRED: command-line for executing your app or a
                             cloud assembly directory (e.g. "node
                             bin/my-app.js")                            [string]
  -c, --context              Add contextual string parameter (KEY=VALUE) [array]
 ...

      --change-set-name      Name of the CloudFormation change set to create

 ...
```

You can use that to differentiate the various deployments.

Thanks,
Adam

**Comment by falnyr** [comment]
Hey @skinny85, this solution works, thanks!
